.github/workflows/linter.yml

```diff
@@ -274,3 +274,46 @@ jobs:
           command: test
           # only run the library tests, no doc tests
           args: --manifest-path ${{ matrix.lib-crates }} --target=${{ matrix.target }} --features=${{ matrix.features }} --lib
+
+  test-examples:
+    name: Test examples
+    runs-on: ubuntu-latest
+    needs: rustfmt
```

Comment:
I think this should also require all of the clippy and checks.

Reply:
I could add *all* the examples to the `clippy-schema` job (which runs on wasm). Would that be sufficient or what do you mean by "checks"?

Reply:
Sorry, I meant that the test job should only run after the clippy job. So to add to the `needs` field.

Reply:
Ah, I see. I'll update it.